src/cnc/models/config/settings.py

```diff
@@ -215,9 +223,8 @@ class ServerlessServiceSettings(BaseServiceSettings):
     handler: Optional[str] = "function.lambda_handler"
     runtime: Optional[str] = "python3.12"
     secrets_mode: Optional[Literal["arn", "plaintext"]] = "plaintext"
-
     cdn: Optional[ServerlessCDNConfig] = Field(default_factory=ServerlessCDNConfig)
-
+    lite: Optional[bool] = False
```

Comment:
I'm not sure it makes sense to have this attribute here. This should be based on the flavor.